CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
-* @martonsereg @waynz0r @Laci21 @tothandras @turip @gallotamas
+* @martonsereg @waynz0r @Laci21 @gallotamas @nishantapatil3 @shanchunyang0919 @tkircsi
```

Comment:
Feel free to remove me (@Laci21) from this list.
@turip wrote this repo basically, let's ask him at least, if it's okay to remove him.
@sagikazarmark, @pregnor would you like to be CODEOWNERS on this repo?

Reply:
Laci, @turip and @tothandras are not in banzai group so if I add them, CODEOWNER check is not able to verify him

Reply:
I see, okay.